src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/GlobalSearchCriteria.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.services
+
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import java.time.LocalDate
+
+@ApiModel("Search Criteria for Global Prisoner Search")
+data class GlobalSearchCriteria (
+  @ApiModelProperty(required = false, value = "Prisoner identifier, one of prisoner number, book number, booking ID or PNC", example = "A1234AA", position = 1)
+  val offenderNo : String?,
```

Comment:
As offender number has meaning in prison api, but this can be one of several different identifiers, I wonder if this should have a different name?  Maybe offenderReference, offenderIdentifier....?